WordPress/src/test/java/org/wordpress/android/ui/domains/management/purchasedomain/PurchaseDomainViewModelTest.kt

```diff
@@ -126,11 +129,28 @@ class PurchaseDomainViewModelTest : BaseUnitTest() {
     }
 
     @Test
-    fun `WHEN a site is chosen THEN send the GoToExistingSite action event`() = testWithActionEvents { events ->
+    @Suppress("MaxLineLength")
+    fun `WHEN a plans eligible site is chosen THEN send the GoToExistingSitePlans action event`() = testWithActionEvents { events ->
```

Comment:
Thank you for adding the new test cases Matthew 🙇 
The look great and cover the checkout cases logic 🥇